packages/web/src/scss/components/Drawer/_Drawer.scss

```
@@ -0,0 +1,40 @@
+@use '../../tools/typography';
+@use 'theme';
+
+.Drawer {
+    @include typography.generate(theme.$drawer-typography);
+
+    all: unset;
+    position: fixed;
+    inset: 0;
+    z-index: 1;
+    display: flex;
+    width: 100%;
+    max-width: none;
+    height: 100%;
+    max-height: none;
+    border: none;
+    background-color: theme.$drawer-backdrop-background-color;
+    visibility: hidden;
+
+    &::backdrop {
+        background-color: transparent;
+    }
+
+    @media (prefers-reduced-motion: no-preference) {
+        transition-property: visibility, opacity;
+        transition-duration: theme.$drawer-transition-duration;
+    }
+}
+
+.Drawer--left {
+    justify-content: start;
+}
+
+.Drawer--right {
+    justify-content: end;
+}
+
+.Drawer[open] {
+    visibility: visible;
```

If you set CSS variables for panel here, you don't need to mix classes in files (meaning you don't need to modify DrawerPanel using Drawer--left/--right/[open] in DrawerPanel.scss)

```suggestion
    --drawer-translate-x: 0%;
    
    visibility: visible;
```

---

you can do this for all those combined selectors inside DrawerPanel (.Drawer* .DrawerPanel {).

Also please do not forget to add to Drawer the default value `--drawer-translate-x: 0%;`

And also add the token prefix :)